build/make/Makefile.macos

```diff
@@ -119,7 +119,18 @@ clean:
 		-rm -rf astonia_net/target
 
 distrib:
-	tar czf macos_client.tar.gz --exclude='bin/*.dSYM' bin res
+	@echo "Bundling libraries for distribution..."
+	@for binary in bin/moac bin/libastonia_net.dylib; do \
```

Comment:
Same thought here. I can do a follow up this week to get a proper signed dmg for osx, but for now I think it makes more sense to not bundle dependency libs and system libs